odoosync.php

```diff
@@ -185,4 +177,20 @@ function odoosync_civicrm_post($op, $objectName, $objectId, &$objectRef) {
     $lineItem->process();
   }
 
+  //Contact
+  if ($objectName = 'Contact'
+    && ($op == 'create' || $op == 'edit' || $op == 'delete'|| $op == 'trash'|| $op == 'restore')
```

Comment:
what if the contacct email, website , address, or phone get updated, I don't think this check will catch these updates, right ?